packages/hash/frontend/src/blocks/userBlocks.tsx

```diff
@@ -0,0 +1,86 @@
+import { BlockMetadata } from "blockprotocol";
+import produce from "immer";
+import {
+  createContext,
+  Dispatch,
+  SetStateAction,
+  useContext,
+  useEffect,
+  useMemo,
+} from "react";
+import { useLocalstorageState } from "rooks";
+
+export type UserBlock = BlockMetadata & {
```

Comment:
`BlockMetadata` is being extended here to include 2 keys which are required further below.